### PR TITLE
feature: add free run counter in UI

### PR DESCRIPTION
**Pull request by geclos**
image

### CODE REVIEW

File: apps/web/src/components/EditorHeader/index.tsx [hunk 1]
```diff
@@ -234,6 +241,55 @@ export default function EditorHeader({
...
       </FormFieldGroup>
+      {selectedProvider === envClient.NEXT_PUBLIC_DEFAULT_PROJECT_ID && (
+        <div>
+          {freeRunsCount !== undefined ? (
+            <Text.H6 color='foregroundMuted'>
+              You have consumed{' '}
+              <Tooltip
+                trigger={
+                  <strong className='text-primary'>
+                    {freeRunsCount} of 100 daily free runs.
```
Comment (csansoon) [inline apps/web/src/components/EditorHeader/index.tsx:252]: I think there's a global constant for the limit? Use it here too.

**Review by andresgutgon — DISMISSED** [review]

File: apps/web/src/components/EditorHeader/index.tsx (hunk 1 above)
Comment (geclos) [reply apps/web/src/components/EditorHeader/index.tsx:252]: the constant and the number are different (constant is 1k, but we wanna say 100 here)

Comment (geclos) [reply apps/web/src/components/EditorHeader/index.tsx:252]: that constant and this number are different

Comment (csansoon) [reply apps/web/src/components/EditorHeader/index.tsx:252]: What? Why? Isn't it the daily limit for free runs?

Comment (geclos) [reply apps/web/src/components/EditorHeader/index.tsx:252]: talked offline

**Pull request merged**